bootstrap/viaproxy/build.gradle.kts

```diff
@@ -0,0 +1,38 @@
+java {
+    targetCompatibility = JavaVersion.VERSION_17
+    sourceCompatibility = JavaVersion.VERSION_17
+}
+
```

Comment:
```suggestion
```
This is not needed since indra sets the version now; see https://github.com/GeyserMC/Geyser/pull/4231#issuecomment-1773338201 for reference